File: Examples/VisualBasic/AsposePDF/Pages/DeleteParticularPage.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf

Namespace AsposePDF.Pages
    Public Class DeleteParticularPage
        Public Shared Sub Run()
            ' ExStart:DeleteParticularPage
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_Pages()

            ' Open document
            Dim pdfDocument As New Document(dataDir & Convert.ToString("DeleteParticularPage.pdf"))

            ' Delete a particular page
            pdfDocument.Pages.Delete(2)

            dataDir = dataDir & Convert.ToString("DeleteParticularPage_out_.pdf")
            ' Save updated PDF
            pdfDocument.Save(dataDir)
            ' ExEnd:DeleteParticularPage
            System.Console.WriteLine(Convert.ToString(vbLf & "Particular page deleted successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace

File: Examples/CSharp/AsposePDF/Pages/DeleteParticularPage.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;

namespace CSharp.AsposePdf.Pages
{
    public class DeleteParticularPage
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_Pages();

            //open document
            Document pdfDocument = new Document(dataDir + "DeleteParticularPage.pdf");

            //delete a particular page
            pdfDocument.Pages.Delete(2);

            //save updated PDF
            pdfDocument.Save(dataDir + "DeleteParticularPage_out.pdf");

        }
    }
}